doc/source/theory/pQCD.rst

```diff
@@ -92,6 +92,7 @@ evolution, always evaluating the strong coupling at :math:`\mu_R^2`.
         1 - a_s L \gamma^{(0)} + a_s^2 \left [ L \gamma^{(1)} + \frac{1}{2} L^2 \gamma^{(0)} (\gamma^{(0)}+\beta_0) \right ]
 
     In this way the dependence of the |EKO| on :math:`L` is factorized outside the unvaried evolution kernel.
+    This procedure is repeated for each different |FNS| present in the evolution path.
```

Comment:
In some sense even a VFNS is a FNS. Maybe replacing with `|FFNS|` might be better?
I'm not actually sure, do what you prefer.

Reply:
I'd say "scheme" is an overall treatment (meaning a global thing) - why don't we stick to our nomenclature "patch"?

Reply:
Okay patch is fine!